cmake/modules/FindPySpark.cmake

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+#
+# Try to find pyspark
+# Once done this will define
+#
+#  PYSPARK_FOUND - system has pyspark and its dependencies are met
+#
+
+message(STATUS "Looking for pyspark dependency: Java")
```

Comment:
Traditionally, `findXYZ` handle a `REQUIRED` parameter that errors out of `REQUIRED` but not found. This could be set for handling `fail-on-missing`, as done for the other `findPackage` cases.

Reply:
In the next commit `find_package(Java)` will have a `REQUIRED` parameter in all cases, since it's a strict requirement for pyspark that cannot be catched in `import pyspark` unfortunately